.travis.yml

```diff
@@ -1,37 +1,49 @@
-language: python
-sudo: false
-
-services:
-  - mongodb
-
-python:
-  - 2.7
-  - 3.4
-
-before_install:
-  - if [ ${TRAVIS_PYTHON_VERSION:0:1} == "2" ]; then wget http://repo.continuum.io/miniconda/Miniconda-3.5.5-Linux-x86_64.sh -O miniconda.sh; else wget http://repo.continuum.io/miniconda/Miniconda3-3.5.5-Linux-x86_64.sh -O miniconda.sh; fi
-  - chmod +x miniconda.sh
-  - ./miniconda.sh -b -p /home/travis/mc
-  - export PATH=/home/travis/mc/bin:$PATH
-  - export MDS_HOST=localhost
-  - export MDS_DATABASE=test
-  - export MDS_TIMEZONE=US/Eastern
-  - mkdir -p /home/travis/.config/metadatastore
-  - 'echo ''port: 27017'' > /home/travis/.config/metadatastore/connection.yml'
-
-install:
-  - export GIT_FULL_HASH=`git rev-parse HEAD`
-  - conda update conda --yes
-  - conda create -n testenv --yes pip nose python=$TRAVIS_PYTHON_VERSION pymongo six pyyaml numpy pandas jinja2
-  - conda install -n testenv --yes -c nikea mongoengine
-  - source activate testenv
-  - pip install coveralls prettytable humanize boltons codecov
-  - python setup.py install
-
-
-script:
-  - python run_tests.py -v
-
-after_success:
-  - coveralls
-  - codecov
+language: python
+sudo: false
+
+services:
+  - mongodb
+
+matrix:
+  include:
+    - python: 2.7
+    - python: 3.4
+    - python: 3.5
```

Comment:
it is here because some of the deps are not yet built on 3.5 and I didn't feel like building them yet.